robottelo/ui/base.py

```diff
@@ -186,7 +186,7 @@ def delete_entity(self, name, really, name_locator, del_locator,
         else:
             raise Exception("Could not search the entity '%s'" % name)
 
-    def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
+    def wait_until_element(self, locator, timeout=20, poll_frequency=0.5):
```

Comment:
Are you really sure that you want to change that top level waiter value as it will affect all ui tests? You can change that value locally just if you have slower system, but it is not good practice to change that variable globally.